src/app/shared/components/filters-list/searchbar/searchbar.component.html

```diff
@@ -12,8 +12,13 @@
       [matAutocomplete]="autoSearch"
       (keyup.enter)="onValueEnter()" />
     <mat-autocomplete #autoSearch="matAutocomplete" class="search-input" panelClass="dropdown-panel" (optionSelected)="onValueSelect()">
-      <mat-option class="dropdown-option option" *ngFor="let result of filteredResults" [value]="result">
-        {{ result }}
+      <mat-option class="dropdown-option option search-dropdown-option" *ngFor="let result of filteredResults" [value]="result">
+        <span class="option-value">{{ result }}</span>
+        <mat-icon
```

Comment:
One possible way to solve the reliability problem is to use a button instead of an icon, try using mat-icon-button insted.
If this doesn't help, I'm fine with leaving it as it is.

Reply:
Even with mat-icon-button pressing Tab to select the previous search result (and its close button) causes all search results to close.
So we leave it.